.github/workflows/cocoapods-lint.yml

```diff
@@ -3,27 +3,32 @@ name: CocoaPods Lint
 on:
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
 jobs:
   lint:
     runs-on: macos-latest
+    timeout-minutes: 10
     strategy:
       fail-fast: false
       matrix:
         include:
           - platform: iOS
-            config: ''
+            config: ""
           - platform: iOS
-            config: '--use-libraries'
+            config: "--use-libraries"
           - platform: iOS
-            config: '--use-libraries --use-modular-headers'
+            config: "--use-libraries --use-modular-headers"
           - platform: iOS
-            config: '--use-static-frameworks'
+            config: "--use-static-frameworks"
           - platform: macOS
-            config: ''
+            config: ""
           - platform: tvOS
-            config: ''
+            config: ""
           - platform: watchOS
-            config: ''
+            config: ""
```

Comment:
@GLinnik21 FYI: These changes are generated by VSCode auto-format-on-save. Hope you don't mind